storage/innobase/include/ut0new.h

```diff
@@ -568,14 +619,18 @@ class ut_allocator {
 	objects of type 'T' and trace the allocation.
 	@param[in]	n_elements	number of elements
 	@param[out]	pfx		storage for the description of the
-	allocated memory. The caller must provide space for this one and keep
+	allocated memory.
+	@param[in]      dontdump        if true, advise the OS is not to core
+	dump this memory.
+        The caller must provide space for this one and keep
```

Comment:
I believe that the `@param[in] dontdump` line should be added immediately before the `@return`. The comment of `pfx` is several lines long.